packages/mwp-api-proxy-plugin/src/util/receive.js

```diff
@@ -255,8 +256,10 @@ export const makeReceive = request => {
 	const injectResponseCookies = makeInjectResponseCookies(request);
 	return query => {
 		const parseApiResponse = makeParseApiResponse(query);
-		const apiResponseToQueryResponse = makeApiResponseToQueryResponse(query);
-		return response => {
+		const apiResponseToQueryResponse = makeApiResponseToQueryResponse(
+			query
+		);
+		return newrelic.createTracer('meetupApiRequest', response => {
```

Comment:
the only question i had was more around this, which looks to be tracing after the request resolves, do we have one in other PR for when it began?

Reply:
yeah the lifecycle of `createTracer` is a little awkward to reason about, but I think this is correct - the trace timer starts when `createTracer` is called, and it finishes when the callback is called. In this case, `createTracer` on line 262 is called when the receiver is first initialized - which happens just _before_ the requests are sent. The callback is called when the responses are _received_. I'll have another look at this to see if there's a better way of composing send & receive to make the lifecycle more clear, but if it's a significant refactor I'll probably leave as-is for now.